packages/fast-development-site-react/src/components/site/title.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import manageJSS, { ComponentStyles, IJSSManagerProps, IManagedClasses } from "@microsoft/fast-jss-manager-react";
+import { IDevSiteDesignSystem } from "../design-system";
+
+
+export interface ISiteTitleProps {
+    slot?: string;
+}
+
+export interface ISiteTitleBrandedManagedClasses {
+    titleAcronym: string;
```

Comment:
Do we want to pick something a bit more extensible? It's an acronym for our use case, but this may not be the same for other teams or users. Perhaps we could do a sync tomorrow AM and land something?

Reply:
It's technically a "branded" string, so perhaps we want to go with something generic like that?

Reply:
resolved